src/kudu/consensus/raft_consensus.cc

```diff
@@ -3761,6 +3761,102 @@ Status RaftConsensus::GetVoterDistribution(std::map<std::string, int32> *vd) con
   return cmeta_->voter_distribution(vd);
 }
 
+Status RaftConsensus::ChangeQuorumTYpe(QuorumType type) {
```

Comment:
nit: not TYpe, but Type

Reply:
Was fixed in the next PR, but forgot to update this one.